029-IoTEdge/Student/Challenge-01.md

```diff
@@ -0,0 +1,29 @@
+# Challenge 1: Deploy IoTHub/Edge
+
+**[Home](../README.md)** - [Next Challenge >](./Challenge-02.md)
+
+## Pre-requisites 
+
+- An Azure IoT Hub created in **one of the regions where Device Streams are available**. At the time of writing, these are: Central US, Central US EUAP (Early Updates Access Program), North Europe, and Southeast Asia. Check the updated list here [Device Streams: Regional availability](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-device-streams-overview#regional-availability).
+
+## Introduction
+ This is the first step of getting an environment setup to work from. This will be a straightforward challenge to ensure you have your subscription setup and to devise a plan to work with your team if you're working as part of a group.  This will be used to work as the starting point for IoT Edge communication in subsequent modules. 
+
+## Description
+In this challenge you will establish all necessary cloud components needed to deploy an Edge device.  Consideration should be given to using automated deployment/provisioning capabilities of Azure IoT hub as a means to gain insight into activity occurring within the plant.  
+
+## Success Criteria
+1. IoT Hub created
```

Comment:
Please make success criteria here and in the other challenges a regular, unordered bullet list. Unless they have to be completed in a certain order let's leave the numbers out of it.